README.md

```diff
@@ -1 +1,24 @@
-# blog-app
```

Comment:
- [ ] OPTIONAL
Your and ReadMe is not descriptive and incomplete. Kindly note that ReadMe is mostly the first place that visitors of this project's repository will go and therefore it has to be very detailed, complete, and reflect this milestone/ project. . Kindly ensure to improve upon your ReadMe

Reply:
Hi @rloterh, thanks for pointing it out. I will definitely work on that before moving to the next milestone